source/upgrade.txt

```diff
@@ -47,9 +47,36 @@ dialog.
    :figwidth: 600px
    :alt: Privacy settings
 
-|compass| does not install any updates without your explicit permission.
-Instead, |compass-short| prompts you to upgrade when a new version is
-available.
+If you enable auto updates and have not downgraded |compass|,
+|compass-short| automatically updates the application four hours after
+MongoDB releases new versions. The update does not require your explicit
+confirmation. After your machine finishes the update, |compass-short|
+shows a toast suggesting that you restart the application to apply the
+update. |compass-short| does not apply the version update until you
+restart the application. 
+
+If you enable auto updates and have downgraded |compass| (you installed
+|compass| and then manually installed an earlier version),
+|compass-short| does not automatically update when MongoDB releases a
+new version. If a newer version is available, |compass-short| asks you
+to confirm that you want to install the update through a notification.
+You can dismiss the notification and refrain from updating. 
+
+If a new update is available and you want to update |compass-short|
+prior to the four hour delay, you can click :guilabel:`Check for
```

Comment:
Similarly here, the 4 hour things is a different timer and I just wouldn't mention it at all.